src/chapters/inheritance/testing-inheritance.rst

```diff
@@ -0,0 +1,97 @@
+.. _testinginheritance:
+
+Testing Inheritance
+===================
+
+Not sure you get the whole inheritance idea? Still not sure which fields and methods get inherited and 
+which are overridden? Looking to *test* your understanding?
+
+Knowing what we know now about :ref:`unit-testing` and :ref:`inheritance`, we can test that our 
+subclasses extend their base classes.
+
+We can add a ``CatTests`` project to our ``Cats`` solution and write some code to ensure that 
+``HouseCat`` inherits what we expect it to.
+
+.. sourcecode:: csharp
+   :linenos:
+
+   [TestMethod]
+   public void InheritsBaseInFirstConstructor()
+   {
+      HouseCat keyboardCat = new HouseCat("Keyboard Cat", 7);
+      Assert.AreEqual(7, keyboardCat.Weight, .001);
+   }
+
+Here, we're testing that one of our ``HouseCat`` constructors will call the ``Cat`` constructor
+and appropriately assign the ``HouseCat`` object's ``Weight`` property. Remember, we don't need
+to write unit tests for getters or setters unless they do something extra in addition to getting
+or setting the field. The purpose of this test, though, is less to test getting ``keyboardCat.Weight`` 
+and more to validate that the subclass constructor has inherited the base class constructor.
+
+It's a good practice to test your subclasses to verify the items that they inherit or override.
+
+Check Your Understanding
+------------------------
+
+.. admonition:: Question
+
+   Fill in the blank to test that the default constructor of ``Cat`` is called when the second 
```

Comment:
chris cautioned me once against using default constructor when referring to the no-arg constructor - there is apparently a more basic default cons. on all objects

Reply:
Done!